scalafix-interfaces/src/main/java/scalafix/interfaces/ScalafixEvaluation.java

```diff
@@ -4,19 +4,31 @@
 
 public interface ScalafixEvaluation {
     /**
-     * @return boolean true if scalafix has successfully been evaluated on all files configured and no ScalafixError has been found
+     * @return boolean true if scalafix has successfully been evaluated on all files configured with no error.
      */
     boolean isSuccessful();
 
-    ScalafixError[] getErrors();
+    Optional<EvaluationError> getError();
 
-    Optional<String> getMessageError();
+    /**
+     * @deprecated  replaced by {@link #getError()}
+     */
+    @Deprecated ScalafixError[] getErrors();
+    /**
+     * @deprecated  replaced by {@link #getErrorMessage()}
+     */
+    @Deprecated Optional<String> getMessageError();
+
+    /**
+     * @return a more detailed error message that can be
+     */
+    Optional<String> getErrorMessage();
```

Comment:
I think the signature talks for itself now
```suggestion
    Optional<String> getErrorMessage();
```